config/jobs/kubernetes/sig-testing/apidiff.yaml

```
@@ -43,7 +43,7 @@ presubmits:
         # We get that from "git merge-base".
         #
         # -b can be used more than once.
-        - "./hack/apidiff.sh -r $(git merge-base ${PULL_BASE_SHA} ${PULL_PULL_SHA}) -t ${PULL_PULL_SHA} -b /workspace/sigs.k8s.io/controller-runtime"
+        - "./hack/apidiff.sh -r $(git merge-base ${PULL_BASE_SHA} ${PULL_PULL_SHA}) -t ${PULL_PULL_SHA} -b ${GOPATH}/src/sigs.k8s.io/controller-runtimer"
```

```suggestion
        - "./hack/apidiff.sh -r $(git merge-base ${PULL_BASE_SHA} ${PULL_PULL_SHA}) -t ${PULL_PULL_SHA} -b ${GOPATH}/src/sigs.k8s.io/controller-runtime"
```

I think you added a trailing `r` by mistake

---

There goes my chance for a fifth fixup PR! Darn, my cunning plan has been foiled.